cypress/support/types/yves/locale.ts

```diff
@@ -0,0 +1,7 @@
+import { Locale } from './shared';
+
+export interface LocaleStaticFixtures {
+  localeDE: string;
+  localeEN: string;
+  availableLocales: Locale[];
```

Comment:
```suggestion
  availableLocales: string[];
```
based on your json file